src/main/java/org/dependencytrack/persistence/jdbi/DefineApiProjectAclCondition.java

```diff
@@ -48,12 +48,12 @@
     /**
      * @return Name of the attribute to define the condition as
      */
-    String name();
+    String name() default JdbiAttributes.ATTRIBUTE_API_PROJECT_ACL_CONDITION;
 
     /**
      * @return Alias of the {@code PROJECT} table to use in the condition
      */
-    String projectTableAlias();
+    String projectIdColumn();
```

Comment:
Changed this to accept a full column name, so the condition also works *without* having to join the `PROJECT` table first. See `ComponentDao#isAccessible` for example, where we can use the `PROJECT_ID` column directly.